src/mugs.js

```diff
@@ -573,7 +574,7 @@ define([
             }
 
             if (!itextItem || lang === '_ids') {
-                return this.getNodeID();
+                return '[' + nodeID + ']';
```

Comment:
What do you think about *not* displaying brackets when `lang === '_ids'`? Similar to displaying the default language in brackets when the selected translation is empty, the node ID is wrapped in brackets only when it is used as a fallback because the labels are empty. This minimizes visual noise when the display is set to _ids.